### ISSUE TITLE
Authentication Error on IPadOs

### ISSUE THREAD

**Issue by rileydes** [AppFlowy-IO/AppFlowy#4544]
### Bug Description

Ran into an authentication error running the testflight app on ipados. Made another fresh account associated with a google account on windows on release 0.4.3 using the builtin sync, made some files. Then tried logging in on IpadOS and and throws errors about submit failing. But logging in with a different google account never associated with appflowy works on ipados.

### How to Reproduce

New account created on windows in 0.4.3 using AppFlowy Cloud and sign in with google. Made around 20 files. Then attempted to sign in with google on IOS app from testflight.

### Expected Behavior

I expected to be able to log in.

### Operating System

IPadOS

### AppFlowy Version(s)

0.4.3

### Screenshots

![IMG_0148](https://github.com/AppFlowy-IO/AppFlowy/assets/114444261/31e72593-59fb-4977-9de4-13839db15541)
![IMG_0147](https://github.com/AppFlowy-IO/AppFlowy/assets/114444261/39a0b2c7-6ac3-4c82-8663-243a9894bb02)

### Additional Context

First the apply submit error happened, then on next launch, the empty workspace id happened.

**Comment by appflowy** [comment]
Hi @rileydes. I can't reproduce this issue. Cloud you mind sending me the log files? Please feel free to DM me in Discord.

**Comment by rileydes** [comment]
Is there an easy way to pull non-crash logs out of a test flight app? Or should I try to reproduce the error and send the full sysdiagnose logs?  I don't have a Mac to compile the project for iPad so I can't get logs right from terminal.

**Comment by appflowy** [comment]
> Is there an easy way to pull non-crash logs out of a test flight app? Or should I try to reproduce the error and send the full sysdiagnose logs? I don't have a Mac to compile the project for iPad so I can't get logs right from terminal.

Understood. We need to implement a feature to retrieve log files from the mobile app. Are you able to use the desktop version of AppFlowy?

**Comment by rileydes** [comment]
Yes, it works well. Just a note on the logs feature, it has to be accessible from the splash screen or authentication rather than the settings menu to be useful in diagnosing bugs like these.

**Comment by LucasXu0** [comment]
@rileydes In the upcoming v0.4.4, we've implemented the 'Upload Log Files' feature, allowing users to upload log files directly from the settings page, accessible via the startup or home page.

**Comment by rileydes** [comment]
[appflowy_logs.zip](https://github.com/AppFlowy-IO/AppFlowy/files/14141502/appflowy_logs.zip)
@LucasXu0 @appflowy Tested the new release, same issue. Here are the logs for the ipad authentication bug. The only thing that might be different about my setup than default installation on ipad is that I run Brave as a browser over safari.
